tst/test_plateau_stopper.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from syne_tune.optimizer.baselines import RandomSearch
+from syne_tune import Tuner
+from syne_tune.stopping_criterion import PlateauStopper
+from syne_tune.config_space import randint
+from syne_tune.util import script_height_example_path
+from tst.util_test import temporary_local_backend
+
+
+def test_plateau_scheduler():
```

Comment:
This is more of a integration/smoke test. It would be nice to have a unit-test in addition that tests more cases with artificial cases. This can be done by just showing artificial results (updating a tuning status) to the plateau and checking that the right behavior (stop or not stop) is obtained for patience/mode/top values.

It is very easy to have an error sneaking in otherwise (for instance with some combination of mode/patience).